swap/src/asb/command.rs

```diff
@@ -33,13 +33,13 @@ where
             env_config: env_config(testnet),
             cmd: Command::Start { resume_only },
         },
-        RawCommand::History => Arguments {
+        RawCommand::History { only_unfinished } => Arguments {
```

Comment:
what about `only_active` or `active_only` ? not to bikeshed the name, feel free to ignore

Reply:
No I see your point.
"Unfinished" is a term that is used somewhere in the code base I believe. That's why I chose it.